commands/cli.js

```diff
@@ -26,11 +28,24 @@ export function normalizeFolderPath(folderPath) {
   return path.normalize(folderPath.endsWith(path.sep) ? folderPath.slice(0, -1) : folderPath)
 }
 
-export function getFiles(folderPath) {
+export function getFiles(folderPath, isGitFolder=false) {
+
   let files = []
+  let filters = []
+  
+  // if the folder is git folder then we should ignore 
+  // .git and files / folders in gitignore
+  if (isGitFolder){
```

Comment:
We might do the reverse operation.
Instead of expecting this flag, we might look at the existence of the .gitignore file or .git folder to not deploy by default the folder attached.
But if we want to deploy the files referenced in the .gitignore, add a flag to explicit it (`--include-git-ignored-files`) 
In all cases, there is no point to deploy the .git folder

Reply:
> We might do the reverse operation.
> Instead of expecting this flag, we might look at the existence of the .gitignore file or .git folder to not deploy by default the folder attached.
> But if we want to deploy the files referenced in the .gitignore, add a flag to explicit it (`--include-git-ignored-files`)
> In all cases, there is no point to deploy the .git folder

Hadn't seen your comment sorry, should be fixed now. `.git` is always ignored, `.gitignore` files are ignored by default unless  `--include-git-ignored-files` flag is added to the deploy command.